File: from-2201-to-2300/2225-find-players-with-zero-or-one-losses/solution.go
func findWinners(matches [][]int) [][]int {
    players := make(map[int]bool)
    loses := make(map[int]int)
    for _, match := range matches {
        players[match[0]] = true
        players[match[1]] = true
        loses[match[1]] += 1
    }
    lose_zero := []int{}
    lose_one := []int{}
    for p, _ := range players {
        if loses[p] == 0 {
            lose_zero = append(lose_zero, p)
        } else if loses[p] == 1 {
            lose_one = append(lose_one, p)
        }
    }
    sort.Ints(lose_zero)
    sort.Ints(lose_one)
    return [][]int{lose_zero, lose_one}
}

File: from-2201-to-2300/2225-find-players-with-zero-or-one-losses/solution.py
class Solution:
    def findWinners(self, matches: List[List[int]]) -> List[List[int]]:
        loses = dict()
        for winner, loser in matches:
            if winner not in loses:
                loses[winner] = 0
            loses[loser] = loses.get(loser, 0) + 1

        no_lose = sorted([w for w, l in loses.items() if l == 0])
        one_lose = sorted([w for w, l in loses.items() if l == 1])

        return [no_lose, one_lose]